vm/src/obj/objbytes.rs

```diff
@@ -208,3 +209,15 @@ fn bytes_iter(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 
     Ok(iter_obj)
 }
+
+fn bytes_clear(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(zelf, Some(vm.ctx.bytes_type()))]);
+    let mut mut_obj = zelf.borrow_mut();
+    match mut_obj.payload {
```

Comment:
Instead of matching on the payload, you can use the `get_mut_elements` helper, e.g.:

```rust
get_mut_elements(zelf).clear();
```